tests/subsys/fs/littlefs/src/testfs_lfs.c

```diff
@@ -27,7 +27,7 @@ struct fs_mount_t testfs_small_mnt = {
 };
 
 #if CONFIG_APP_TEST_CUSTOM
-FS_LITTLEFS_DECLARE_CUSTOM_CONFIG(medium, MEDIUM_IO_SIZE, MEDIUM_IO_SIZE,
```

Comment:
Shouldn't this be rather size of pointer or int? Do we have some define that shows what is platform arch/platform alignment?